Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```diff
@@ -120,6 +120,29 @@
       params:
         volume: 12
 
+- type: entity
+  name: Self Destruct
```

Comment:
add noSpawn: true

Reply:
got this one too
![image](https://github.com/space-wizards/space-station-14/assets/156087924/830b729e-d05c-49fe-83b3-86c52295a754)